torch/csrc/autograd/init.cpp

```diff
@@ -52,10 +56,22 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject *unused) {
   m.def("_enable_profiler", enableProfiler);
   m.def("_disable_profiler", disableProfiler);
   m.def("_profiler_enabled", profilerEnabled);
-  m.def("_run_before_callbacks", _runBeforeCallbacks);
 
-  py::class_<RecordFunction, std::shared_ptr<RecordFunction>>(m, "_RecordFunction")
-    .def(py::init<>());
+  // TODO: remove when jit future can hold PyObject (https://github.com/pytorch/pytorch/issues/34999)
+#ifdef USE_DISTRIBUTED
```

Comment:
Needed this to fix the windows build since it doesn't build with dist, let me know if this isn't the best way